framework/managers/MqttConnectionManager.cpp

```diff
@@ -61,8 +61,12 @@ void MqttConnectionManager::onDisconnected(TcpClient &client, bool flag) {
         LOG.log("Unreachable");
     }
 
-    this->state.set(State::DISCONNECTED);
-    this->reconnectTimer.start();
+    if (this->getState() != State::CONNECTING) {
```

Comment:
You can use `this->state` here as it casts to the wrapped value

Reply:
Totally missed that... :D